x/ibc/05-port/keeper/keeper.go

```diff
@@ -60,5 +60,9 @@ func (k Keeper) Authenticate(key sdk.CapabilityKey, portID string) bool {
 		panic(err.Error())
 	}
 
+	if key.Name() != portID {
+		return false
+	}
+
```

Comment:
I think there should add some complementary check, or panic?  @fedekunze @AdityaSripal

Reply:
What do you mean by complementary check?

Reply:
On `BindPort`, the key.Name() is equal to the portID. So I think we should add this check, it is what the test case expects, refer https://github.com/cosmos/cosmos-sdk/blob/5d4b1e37396ed584d4af821d7582f03ca115e9a2/x/ibc/05-port/keeper/keeper_test.go#L64.